src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditEntityExactlyOne.razor.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class EditEntityExactlyOne : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public GenericEntityModel? Entity { get; set; }
+
+    [Parameter]
+    public string? Id { get; set; }
+
+    public string? CurrentIdStr { get; set; }
```

Comment:
Maybe we can use Id, rename it to CurrentId and have only one property for this. I think CurrentIdStr is redundant.

Reply:
within latest commit